oceanbench/evaluate.py

```diff
@@ -2,13 +2,71 @@
 #
 # SPDX-License-Identifier: EUPL-1.2
 
-from oceanbench.core.python2jupyter import (
-    generate_evaluation_notebook_file,
-)
+from enum import Enum
+from os import environ
 
+from oceanbench.core import evaluate
 
-def generate_notebook_to_evaluate(path_to_canditate_python_code_file: str, output_notebook_file_path: str):
-    return generate_evaluation_notebook_file(
-        python_file_path=path_to_canditate_python_code_file,
-        output_notebook_file_path=output_notebook_file_path,
+
+class OceanbenchEnvironmentVariable(Enum):
+    OCEANBENCH_CHALLENGER_PYTHON_CODE_URI_OR_LOCAL_PATH = "OCEANBENCH_CHALLENGER_PYTHON_CODE_URI_OR_LOCAL_PATH"
+    OCEANBENCH_OUTPUT_NOTEBOOK_FILE_NAME = "OCEANBENCH_OUTPUT_NOTEBOOK_FILE_NAME"
+    OCEANBENCH_OUTPUT_BUCKET = "OCEANBENCH_OUTPUT_BUCKET"
+    OCEANBENCH_OUTPUT_PREFIX = "OCEANBENCH_OUTPUT_PREFIX"
+
+
+def _parse_variable_environment(
+    variable: str | None,
+    environment_variable_name: OceanbenchEnvironmentVariable,
+) -> str | None:
+    return variable if variable else environ.get(environment_variable_name.value)
+
+
+def _parse_input_non_manadatory(
+    variable: str | None,
+    environment_variable_name: OceanbenchEnvironmentVariable,
+) -> str | None:
+    return _parse_variable_environment(variable, environment_variable_name)
+
+
+def _parse_input_manadatory(
+    variable: str | None,
+    environment_variable_name: OceanbenchEnvironmentVariable,
+) -> str:
+    parsed_variable = _parse_variable_environment(variable, environment_variable_name)
+    if not parsed_variable:
```

Comment:
```suggestion
    if parsed_variable !== None:
```
Might be better to deal with edge case where the variable could be "False" for example

Reply:
better?